week3/assignment/week3_GayeonKim/src/components/Game/Game.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState, useEffect } from 'react'
+import styles from './Game.module.css';
+
+
+//App에서 넘어오는 두 값, startGame과 stopGame
+const Game = ({startGame, stopGame, playTime}) => {
+    const [numbers, setNumbers] = useState([]);
+    const [currentNumber, setCurrentNumber] = useState(1);
+    const [isRunning, setIsRunning] = useState(false);
+    const [availableNumbers, setAvailableNumbers] = useState([]);
+
+    // const clickTheNumber = (newCount) => {
+    //     setCount(newCount); // view 상태를 클릭한 버튼에 따라 변경
+    //     // console.log(`view state changed! ${view}`)
+    //   };
+
+    //처음 랜덤으로 배치될 1~9
+    useEffect(() => {
+        setNumbers(shuffleArray([1, 2, 3, 4, 5, 6, 7, 8, 9]));
+        setAvailableNumbers(shuffleArray([10, 11, 12, 13, 14, 15, 16, 17, 18]));
+    }, []);
```

Comment:
```suggestion
    useEffect(() => {
setNumbers(shuffleArray(Array.from({ length: 9 }, (_, i) => i + 1)));
    setAvailableNumbers(shuffleArray(Array.from({ length: 9 }, (_, i) => i + 10)));
    }, []);
```

이런식으로 동적 배열을 생성하면 숫자를 반복적으로 나열하지 않아도 배열을 생성할 수 있어요 !